assets/src/edit-story/components/panels/layer/layerList.js

```diff
@@ -45,6 +47,18 @@ const LayerSeparator = styled(ReorderableSeparator)`
   padding: ${LAYER_HEIGHT / 2}px 0;
 `;
 
+const LayerScroller = styled(ReorderableScroller)`
+  height: ${LAYER_HEIGHT}px;
```

Comment:
Let's pass this as a prop of `<Reorderable>`, e.g. `<Reorderable getItemSize={() => LAYER_WIDTH} />` inspired by https://reactnative.dev/docs/flatlist#getitemlayout

Reply:
Fixed in b52fb791.